src/web-auth/index.js

```diff
@@ -484,6 +486,9 @@ WebAuth.prototype.authorize = function(options) {
 
   params = this.transactionManager.process(params);
 
+  var connection = params.realm || params.connection;
+  storage.setItem('auth0.ssodata.connection', connection);
+
   windowHelper.redirect(this.client.buildAuthorizeUrl(params));
```

Comment:
we are doing this in several places, maybe we could have a class that encapsulates reading/setting the localStorage data for SSO?

Reply:
done